tests/fixtures/NoEmptyStatementsLinter/empty_statements.php.autofix.expect

```diff
@@ -0,0 +1,25 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+function foo(): void {
+  some_function();
+  
+
+  // leading comment
+  
+
+   // trailing comment
```

Comment:
sorry for the slow review; are you able to fix this indentation reasonably?

Reply:
No worries!

I'm not sure what the best approach is here so I'm very open to input. What's shown here is the result of retaining all the leading and trailing whitespace around the statement (option #2 in the initial PR description).

I can't think of something simple that gives you good output for all of these scenarios:

```Hack
function whitespace_options(): void {
  ; // trailing comment
  ;   // trailing comment
  ;
}
```

Do you have any ideas in mind?

Reply:
Let's leave this as-is; several linters/migrators are now handling this in their own way - it's probably best fixed by adding a generic "fix the whitespace for this trivia" helper (`hackfmt`-style, not neccessarily trying to preserve it) and replacing all the existing things with it.